src/main/java/core/basesyntax/MainApp.java

```diff
@@ -2,4 +2,15 @@
 
 public class MainApp {
 
+    public static void main(String [] arg) {
+        Truck tru = new Truck();
+        Bulldozer bul = new Bulldozer();
+        Excavator exc = new Excavator();
```

Comment:
<img width="1039" alt="Screenshot 2022-01-17 at 17 26 21" src="https://user-images.githubusercontent.com/33026471/150296399-46afbd78-be61-4106-b2a0-9e9e7eee66bf.png">

and let's fix naming: tru -> truck, etc.